src/name/mlopatkin/andlogview/MainFrame.java

```diff
@@ -343,9 +353,7 @@ private void bindKeyGlobal(String key, String actionKey, Action action) {
 
     private static final String KEY_HIDE_AND_START_SEARCH = "ENTER";
     private static final String KEY_HIDE = "ESCAPE";
-    private static final String KEY_SHOW_SEARCH_FIELD = "control F";
-    private static final String KEY_FIND_NEXT = "F3";
-    private static final String KEY_FIND_PREV = "shift F3";
+    private static final int KEY_FIND_NEXT_PREV = SystemUtils.IS_OS_MACOS ? KeyEvent.VK_G : KeyEvent.VK_F3;
```

Comment:
See my other suggestion to convert these constants to KeyStrokes. The `KEY_FIND_NEXT_PREV` is a bit misleading.